scripts/build/tauri-rename-linux.sh

```diff
@@ -5,24 +5,16 @@
 set -euox pipefail
 
 # For debugging
-ls "$TARGET_DIR/release" "$TARGET_DIR/release/bundle/deb"
+ls "$TARGET_DIR/release" "$TARGET_DIR/release/bundle/deb" "$TARGET_DIR/release/bundle/rpm"
 
-# Used for release artifact
 # In release mode the name comes from tauri.conf.json
-# Using a glob for the source, there will only be one exe and one deb anyway
-cp "$TARGET_DIR/release/firezone-client-gui" "$BINARY_DEST_PATH"
-cp "$TARGET_DIR/release/firezone-gui-client.dwp" "$BINARY_DEST_PATH.dwp"
-cp "$TARGET_DIR/release/bundle/deb/firezone-client-gui.deb" "$BINARY_DEST_PATH.deb"
-cp "$TARGET_DIR/../gui-client/firezone-client-gui.rpm" "$BINARY_DEST_PATH.rpm"
-# TODO: Debug symbols for Linux
+# Using a glob for the source, there will only be one deb anyway
+cp $TARGET_DIR/release/bundle/deb/Firezone*.deb "$BINARY_DEST_PATH.deb"
+cp $TARGET_DIR/release/bundle/rpm/Firezone*.rpm "$BINARY_DEST_PATH.rpm"
```

Comment:
Changing the `productName` also changes what the final package is called. We rename it here anyway so we just need to change the way we glob for it.